reactjs-dallas/reactjs-dallas-site
# chore: update dependencies to latest minor version

# The issue

We are getting a lot of "vulnerability" warnings from GitHub about outdated packages

# The solution

- Upgrade packages to latest minor version
  - Need to do more, but the transition to Gatsby v3 needs to be its own PR work
- Add two new scripts for some handy helpers and update a third one:
  - `dev`: runs the app (used to be `develop`)
  - `clean:gatsby`: runs `gatsby clean`
  - `clean:purge`: removes all node modules then runs 'yarn cache clean`
- Require node `12.x`

# Visuals

Before | After
----- | -----
![image](https://user-images.githubusercontent.com/11624407/125542121-055288c6-b678-4a35-b4bd-1525b9ba07ad.png) | ![image](https://user-images.githubusercontent.com/11624407/125542145-14dd0b12-209b-4552-976d-fb2b004ce0ab.png)

# Test cases

0. Make sure you are on node version `12.x`
1. Get this code
1. Run `yarn`
2. Run `yarn dev` and make sure the app builds correctly
